template/src/components/Screen.tsx

```diff
@@ -23,7 +22,7 @@ interface ScreenProps {
   /** The content to render within the screen */
   children?: ReactNode;
   /** Whether to force the topInset. Use to prevent screen jank on tab screens */
-  forceTopInset?: Boolean;
```

Comment:
noice

Reply:
@dayhaysoos I'm assuming `react-navigation` fixed something here and this is no longer required?

Reply:
@cball I actually have no idea why that went missing. I think @isaiahgrey93  knows why it's gone